src/components/canvas-tools/geometry-tool.tsx

```diff
@@ -205,4 +205,21 @@ class GeometryToolComponent extends BaseComponent<IProps, IState> {
   }
 }
 
-export default sizeMe()(GeometryToolComponent);
+export default class GeometryToolComponent extends React.Component<IProps, {}> {
+
+  public static getDragImageNode(dragTargetNode: HTMLElement) {
+    // dragTargetNode is the tool-tile div
+    // firstChild is SizeMe div
+    const child = dragTargetNode.firstChild;
+    // firstChild's firstChild is the actual SVG, which works as a drag image
+    return child && child.firstChild;
+  }
+
+  public render() {
+    return (
+      <SizeMe>
+        {() => <GeometryToolComponentImpl {...this.props} />}
```

Comment:
The `SizeMe` documentation suggests that it accepts a component here, i.e. we should just be able to say:
```javascript
<SizeMe>
  <GeometryToolComponentImpl {...this.props} />
</SizeMe>
```
but the SizeMe code as currently written requires a function here.

Reply:
That is odd, maybe worth adding as a comment